arrows/vxl/CMakeLists.txt

```diff
@@ -59,8 +59,8 @@ kwiver_add_library( kwiver_algo_vxl
   ${vxl_sources}
   )
 target_link_libraries( kwiver_algo_vxl
-  PUBLIC               kwiver_algo
-                       vil vpgl vgl
+  PUBLIC               kwiver_algo_core
+                       vil vidl vgl
```

Comment:
This library should not change from `vpgl` to `vidl`.  I recently corrected this in another branch.  We need `vpgl` as a public dependency because `vpgl_perspective_camera` is used in some of the public headers.  `vidl` is already listed as a private dependency and shouldn't need to be public.